cocos/asset/assets/image-asset.ts

```diff
@@ -523,43 +485,52 @@ export class ImageAsset extends Asset {
     @override
     get _nativeAsset (): any {
         // Maybe returned to pool in webgl.
-        return this._nativeData;
+        return this._imageData.data;
     }
     // TODO: Property 'format' does not exist on type 'ImageBitmap'
     // set _nativeAsset (value: ImageSource) {
     set _nativeAsset (value: any) {
-        if (!(value instanceof HTMLElement) && !isImageBitmap(value)) {
-            value.format = value.format || this._format;
-        }
         this.reset(value);
     }
 
+    /*
+     * @en Raw image data.
+     * @zh 原始图像数据。
+     */
+    get rawData (): any {
```

Comment:
What's the difference between `rawData` and `data`?

Reply:
There are three data types:

‘rawData’: data containing the content of the image

'data': the source of the image, including HTMLImageElement, HTMLCanvasElement, etc. This may be better to use dataSource, but because this interface is exported, it may affect compatibility issues

'imageData': image data management object, similar to _nativeAsset, but modifying _nativeAsset may lead to compatibility problems, so the imageData interface is added

Reply:
Can combine data and imageData? Only use one interface.

Reply:
You mean to get data through imageData？But the data interface is already an exported interface. Do I need to deprecate this interface?

Reply:
Can just use data?

Reply:
yeh，load images with loadImage removes the imagedata dependency.